tests/settings.spec.js

```diff
@@ -499,21 +499,21 @@ describe('checkInput', () => {
 		const options = {
 			flag1: {
 				type: 'string',
-				arguments: ['DOMINIK', 'THOMAS'],
+				arguments: ['dominik', 'thomas'],
 			},
 			flag2: {
 				type: 'string',
-				arguments: ['DOMINIK', 'THOMAS'],
+				arguments: ['dominik', 'thomas'],
 			},
 			flag3: {
 				type: 'string',
-				arguments: ['DOMINIK', 'THOMAS'],
+				arguments: ['dominik', 'thomas'],
 			},
 		};
 
-		expect(checkInput({ flag1: 'DOMINIK' }, options).pass).toBeTruthy();
```

Comment:
<img width="751" alt="X6ha7U" src="https://user-images.githubusercontent.com/737821/79715241-8d380780-8316-11ea-9e42-9e806914dc8c.png">

I had to change these as uppercase is no longer a thing (correct me if i'm wrong), so I made them all lowercase and swapped around the uppercase check